server.js

```diff
@@ -1,35 +1,88 @@
-import express from "express";
-import cors from "cors";
-import mongoose from "mongoose";
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo";
-mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true });
-mongoose.Promise = Promise;
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
-const port = process.env.PORT || 8080;
-const app = express();
-
-// Add middlewares to enable cors and json body parsing
-app.use(cors());
-app.use(express.json());
-
-// Start defining your routes here
+import express from "express"
+import cors from "cors"
+import mongoose from "mongoose"
+import netflixData from "./data/netflix-titles.json"
+
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+mongoose.set('strictQuery', false);
+mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
+mongoose.Promise = Promise
+
+const port = process.env.PORT || 8080
+const app = express()
+
+app.use(cors())
+app.use(express.json())
+
+const Title = mongoose.model("Title", {
+  show_id: Number,
+  title: String,
+  director: String, 
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String
+})
+
+if(process.env.RESET_DB) {
+  const resetDataBase = async () => {
+    await Title.deleteMany();
+   netflixData.forEach(singleTitle => {
+      const newTitle = new Title(singleTitle)
+      newTitle.save()
+    })
+
+  }
+  resetDataBase()
+}
+
+// Routes
+
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
-});
+  res.send("This is an api for Netflix-titles. Go to /titles to see all titles, and then add the _id of the title in /titles/ to see a specific title")
```

Comment:
Okey here I got the answer to my first question. Nice solution, I will try to add the same :D